doc.html

```diff
@@ -47,104 +47,79 @@ <h2>4 produce_demand_forecaster Usage Scenarios</h2>
-|     Outputs | Supported parameters         |
+|     Outputs | array of top produce         |
+|             | (meta)data                   |
 +-------------+------------------------------+
 </pre>
-Please refer to the openAPI produce_demand_forecaster specification for details about data schemas and formats returned by the capabilities endpoint, as well as to ANNEX 1 for a list of normalized measurements that may be offered by a produce_demand_forecaster service.
-
-produce_demand_forecaster services are not required to handle intense traffic from a single client, such as the one that may result from being directly invoked on user interface interactions in an FMIS, for instance. Implementors MAY generate a 429 TOO MANY REQUESTS error response if the rate of calls exceeds some pre-defined quota. Clients requiring more intense field information are advised to implement a mirroring mechanism using the subscription functions.
-
-<h2>5.1 Queries with no data available</h2>
 
-It could happen that there is no data available for the requested parameters at the specified boundary and time interval. A common example for that could be that sensors for the requested parameters were never installed within the boundaries, or they were not online during the time interval.
+<h2>5.1 Invalid GEO code</h2>
 
-In those cases, an error message will be returned to the request.
+An error code 404 with an error message will be returned to the request.
 
 <h2>5.2 Invalid time interval</h2>
 
-Sometimes the requested time interval may be invalid according to the available sensor data. For example, if the time interval is 7 minutes and only time series of 1 hour are available.
-
-In all those cases an error would be returned to the request.
-
-To solve the most common requests, the next default time intervals should be covered by the produce_demand_forecaster service implementor: 1 Hour, 1 day, 1 week, 1 month and their multiples.
+An error code 400 with an error message will be returned to the request.
 
 <h2>6 Access and Authentication</h2>
 
-Farmers MUST have an account setup on an produce_demand_forecaster service in order to authenticate and access API endpoints. The service implementor is responsible for the creation of accounts; it is not covered in the service template specifications.
+The produce_demand_forecaster is publically available with rate limits.
+
+For less strict rate limits, producers and retailers and  MUST have an account setup on an produce_demand_forecaster service in order to authenticate and access API endpoints. 
 
-Unless specifically documented in the OpenAPI specifications, all API calls must include credentials in form of Bearer authentication (also called token authentication). Clients can obtain an access token on behalf of their user from the service's authorization server (see ATLAS service pairing).
+To obtain credentials that give access to the endpoints, reach out to the Calabi staff
```

Comment:
Please leave the original paragraph. Keep in mind that the service template specifications MUST be vendor and technology agnostic.